### DISCUSSION TITLE
How to reach the tree-sitter community?

### DISCUSSION THREAD

**Discussion by magthe** [tree-sitter/tree-sitter#2155] [Q&A] [answered]
I'm writing a parser for [Cabal](https://cabal.readthedocs.io/en/stable/index.html), Haskell's packaging tool, and I've been looking for a place to reach people working on tree-sitter and its parsers as well as authors of software using tree-sitter, i.e. the "tree-sitter community". Is there such a place?

In particular Cabal is indentation sensitive and I have questions around my approach to handling it (influenced by the Python parser).

I keep my parser [here](https://gitlab.com/magus/tree-sitter-cabal).

**Comment by sogaiu** [comment]
May be this is of interest?  https://github.com/tree-sitter/tree-sitter/discussions/2027

**Comment by ahlinc** [comment, marked as answer]
@magthe you can ask questions here in discussions, many people see such questions and if someone have a time and related knowledge you may got an answer. Also you can join a Discord server (a link is in a pinned discussion in this repo) where you may find other discussions and answers, but I'd suggest to use _Discussions_ in the repo first because it gives more structure for questions. And try to search for answers of your questions  in this repo issues and discussions may be you'll find something useful for you cases.
